packages/edit-post/src/components/layout/index.js

```diff
@@ -66,7 +72,7 @@ function Layout( {
 		tabIndex: -1,
 	};
 	return (
-		<div className={ className }>
+		<FocusReturnProvider className={ className }>
```

Comment:
What happens if I use two of these providers? Say each `BlockEditorProvider` uses one?

Reply:
> What happens if I use two of these providers? Say each `BlockEditorProvider` uses one?

Generally, I'd say it should probably be avoided, if possible. The component should fall under a "one-per-app" recommendation.

That said, in practice it shouldn't be too problematic. Any `withFocusReturn`-enhanced component would rely on its closest provider as a source to determine where focus returns. The only potential problem is that the highest provider would still detect focus changes into an area technically governed by another provider, so they're not mutually exclusive. It's hard to imagine it would result in any unexpected behavior, though.